tensorflow_addons/losses/triplet.py

```diff
@@ -69,7 +69,10 @@ def _masked_minimum(data, mask, dim=1):
 @tf.keras.utils.register_keras_serializable(package="Addons")
 @tf.function
 def triplet_semihard_loss(
-    y_true: TensorLike, y_pred: TensorLike, margin: FloatTensorLike = 1.0
+    y_true: TensorLike,
+    y_pred: TensorLike,
+    margin: FloatTensorLike = 1.0,
+    angular: bool = False,
```

Comment:
just wondering, why isn't "squared" a parameter originally? (for l1 vs l2 distance)?

Reply:
I can only speak to what I have added, but I agree it is odd that there is not an option especially give that the pairwise_dist function accepts squared as an option. I can certainly add this parameter.

Reply:
Or maybe not. Adding the squared parameter is causing a single build release wheel check to fail so maybe I should just revert that commit.